keps/sig-cli/3104-introduce-kuberc/README.md

```diff
@@ -305,21 +313,32 @@ required) or even code snippets. If there's any ambiguity about HOW your
 proposal will be implemented, this is the place to discuss them.
 -->
 
-During alpha this feature will be enabled through the `KUBECTL_KUBERC=true` environment variable. The file will default to being located in `~/.kube/kuberc`. A flag will allow overriding this default location with a path i.e. `kubectl --kuberc /var/kube/rc`.
+For beta this feature will be enabled by default. However users can disable it
+setting the `KUBECTL_KUBERC` environment variable to `false`.
```

Comment:
Users can also disable this functionality via `KUBERC=off` anytime regardless of beta, ga, alpha, etc.

Reply:
Updated.